lib/ovirt/service.rb

```diff
@@ -173,6 +173,7 @@ def probe_api_path(uri, path)
         request.get
       rescue RestClient::Exception => exception
         response = exception.response
+        return false if response.nil?
```

Comment:
This will work. In this case - thinking about having to debug this one day - maybe its a good idea to add a log with the exception? cause it gets swallowed here.